docs/src/main/paradox/release-notes/10.1.x.md

```diff
@@ -1,5 +1,13 @@
 # 10.1.x Release Notes
 
+## 10.1.10
+
```

Comment:
```suggestion

### Migration notes

[RFC 7231](https://tools.ietf.org/html/rfc7231#section-6.3.6) dictates an HTTP response with status code 205 ('Reset Content') is not allowed to have an entity body. Since [#2686](https://github.com/akka/akka-http/pull/2686) we enforce this restriction, so if you (incorrectly) produced such responses you will have to either remove the entity body or select a different status code.

```

Reply:
Thanks for the suggestion :)